packages/framework-provider-local/src/services/read-model-registry.ts

```diff
@@ -32,27 +37,67 @@ export class ReadModelRegistry {
     const queryPromise = new Promise((resolve, reject) =>
       cursor.exec((err, docs) => {
         if (err) reject(err)
-        else resolve(docs)
+        else {
+          docs.map((doc: ReadModelEnvelope) => {
+            doc.value.boosterMetadata!.optimisticConcurrencyValue = this.generateOptimisticConcurrencyValue(doc.value)
+          })
+          resolve(docs)
```

Comment:
Why do we need to do this when reading the read model? Would it not be already stored in the DB?

Reply:
There could be old readmodels where the value is undefined